features/lwipstack/mbed_lib.json

```diff
@@ -120,6 +120,10 @@
             "help": "Stack size for lwip TCPIP thread",
             "value": 1200
         },
+        "tcpip-thread-priority": {
+            "help": "priority of lwip TCPIP thread",
+            "value": 24
```

Comment:
I'd prefer this to be set by name - make it `"osPriorityNormal"`. `24` is a bit vague.

Strings work fine here - a JSON string becomes C token(s). (If you wanted a C string you'd need to do `"\"string\""`).